.github/workflows/vis-reg-tests.yml

```diff
@@ -10,12 +10,24 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/ci-setup
 
+      # only packages have to be built as they might be imported by Storybooks
+      - run: pnpm build --filter='!./apps/*'
+
       - name: Build Storybook for Design System
-        run: pnpm storybook:ds:build
+        run: pnpm turbo run storybook:build --filter=@webstudio-is/design-system
+      - name: Build Storybook for Designer
+        run: pnpm turbo run storybook:build --filter=@webstudio-is/designer
 
-      - name: Lost Pixel
+      - name: Lost Pixel for Design System
         uses: lost-pixel/lost-pixel@v2.23.0
         env:
           S3_ACCESS_KEY: ${{ secrets.S3_ACCESS_KEY }}
           S3_SECRET_KEY: ${{ secrets.S3_SECRET_KEY }}
           LOST_PIXEL_CONFIG_DIR: packages/design-system
+
+      - name: Lost Pixel for Designer
+        uses: lost-pixel/lost-pixel@v2.23.0
+        env:
+          S3_ACCESS_KEY: ${{ secrets.S3_ACCESS_KEY }}
+          S3_SECRET_KEY: ${{ secrets.S3_SECRET_KEY }}
+          LOST_PIXEL_CONFIG_DIR: apps/designer
```

Comment:
```suggestion
      - name: Lost Pixel for Design System
        uses: lost-pixel/lost-pixel@v2.23.0
        env:
          S3_ACCESS_KEY: ${{ secrets.S3_ACCESS_KEY }}
          S3_SECRET_KEY: ${{ secrets.S3_SECRET_KEY }}
          LOST_PIXEL_CONFIG_DIR: packages/design-system

      - name: Lost Pixel for Designer
        uses: lost-pixel/lost-pixel@v2.23.0
        env:
          S3_ACCESS_KEY: ${{ secrets.S3_ACCESS_KEY }}
          S3_SECRET_KEY: ${{ secrets.S3_SECRET_KEY }}
          LOST_PIXEL_CONFIG_DIR: apps/designer
          
      - name: Lost Pixel
        uses: lost-pixel/lost-pixel@v2.24.1
        with:
          FINALIZE: true
```

Reply:
Happy new year guys!

Please tell me when you are ready to work on this & try this out. Then I'd need to flick the monorepo support for you manually & you will need to accept these changes :D

Reply:
Happy new year! Let's try this